__tests__/data/last_day_nearest_eom_445_arbitrary_leap_year_weeks.ts

```diff
@@ -1,4 +1,4 @@
-export const lastDayNearestEOM445ArbitraryLeapYearWeeks = [
+export const lastDayNearestEOM445Weeks = [
```

Comment:
2023 is not a leap year, given that configuration. I am not sure how the tests passed in the first place.
The new unit test introduced in this PR covers, all addLeapWeekToMonth options, so this specific test is not needed to be run on leap year.